Zend/zend_language_parser.y

```diff
@@ -108,6 +109,7 @@ static YYSIZE_T zend_yytnamerr(char*, const char*);
 %token T_YIELD        "yield (T_YIELD)"
 %token T_YIELD_FROM   "yield from (T_YIELD_FROM)"
 %token T_PLUS_EQUAL   "+= (T_PLUS_EQUAL)"
+%token T_PIPE         "|> (T_PIPE)"
```

Comment:
nit: the `%token` list seems grouped by categories (not sorted alphabetically), so here seems "off"... in doubt, I guess it's better to add it at the end (before `T_BAD_CHARACTER`)

Reply:
Moved to the end of the list.  It would be cool if there was a documented order it should use. :smile: